idaes/models_extra/power_generation/costing/power_plant_capcost.py

```diff
@@ -1414,32 +1658,32 @@ def get_variable_OM_costs(self, production_rate, resources, rates, prices={}):
 
         # if costing power make vars in fs.costing, if costing hydrogen make vars
         # in fs.H2_costing
-        # if mode == "power":
-        #     costing = self.costing
-        # elif mode == "hydrogen":
-        #     self.H2_costing = Block()
-        #     costing = self.H2_costing
+        if mode == "power":
```

Comment:
why is this back again?

Reply:
I will revert this back to what it was before, and let you know if any issues come up with the tests.